reference/toolstacks/php/ezplatform/README.md

```diff
@@ -0,0 +1,35 @@
+# Getting started
+
+## Prerequisites
```

Comment:
I don't know. I copied this file verbatim from the Symfony docs. This PR is just a place for the eZ team to push stuff to for their own review.  No need for us to review it yet. :smile: